internal/utils/osinfo_darwin.go

```diff
@@ -0,0 +1,26 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package utils
+
+import (
+	"os/exec"
+	"runtime"
+	"strings"
+
+	"github.com/DataDog/dd-sdk-go-testing/internal/constants"
+)
+
+func OSName() string {
+	return runtime.GOOS
+}
+
+func OSVersion() string {
+	out, err := exec.Command("sw_vers", "-productVersion").Output()
+	if err != nil {
+		return constants.Unknown
```

Comment:
is this done in other tracers? could you not set the tag instead?

Reply:
In the .NET tracer I don't check if is null or empty, so I'm always setting the tag (because I'm expecting to always have a value in the supported platforms). Do you prefer not setting this tag in this case then?

Reply:
I'd prefer not to set the tag personally, but maybe it's something to discuss in the Tracers Guild.
I've seen some other values for when the value is missing like `N/A`